frontend/src/entities/jobs/hpo/create/hpo-job-create.tsx

```diff
@@ -351,7 +354,29 @@ export function HPOJobCreate () {
                     `Successfully created hyperparameter tuning job with name ${state.form.HyperParameterTuningJobName}`,
                     'success'
                 );
-
+                payload.HPOJobDefinition.TrainingJobDefinitions.forEach((definition) => {
+                    const split = definition.OutputDataConfig.S3OutputPath.split('/');
+                    const scope = split[4];
+                    let datasetType = DatasetType.PROJECT;
+                    if (split[3] === 'private') {
+                        datasetType = DatasetType.PRIVATE;
+                    }
+                    const datasetName = split[6].replace('"', '');
+                    const newDataset = {
+                        name: datasetName,
+                        description: `Dataset created as part of the HPO job: ${state.form.HyperParameterTuningJobName}`,
+                        type: datasetType,
+                        scope: scope
+                    } as IDataset;
```

Comment:
Cab we use `datasetFromS3Uri()` here?

Reply:
Good call, wasn't aware of this function but it serves the same purpose.